### ISSUE TITLE
Communication with the Neos server may stop and session information and friend list may not be updated.

### ISSUE THREAD

**Issue by kazu0617** [Neos-Metaverse/NeosPublic#2767]
### Describe the bug?

For the past few days, my Neos friend information has been incorrect, and my active session information has been incorrect as well.
As a temporary workaround, when I click on "Logout" in "Home", I can't "logout" and instead it reconnects with the server, or at least it syncs the friend and session information at once and displays them correctly (I can't logout).

### Relevant issues

idk.

### To Reproduce

1. Launch Neos.
2. Spend some time in Neos until it happens.
3. Check it when you are spending time with more than one person (see the same person as your friends, online status or friend status is old).
4. Trying to logout. Can't do it, but the update is done.

### Expected behavior

Provide some kind of method that can show some kind of problem in Neos or fundamentally check if it is currently up to date.

### Log Files

```
Exception when updating CloudX Interface:
System.InvalidOperationException: Proxy has been unregistered!
  at CloudX.Shared.CloudVariableProxy.WriteToCloud () [0x00009] in <b5c3a2458ce849218fe52886b5494325>:0 
  at CloudX.Shared.CloudVariableManager.Update () [0x00039] in <b5c3a2458ce849218fe52886b5494325>:0 
  at CloudX.Shared.CloudXInterface.Update () [0x000e0] in <b5c3a2458ce849218fe52886b5494325>:0 
  at FrooxEngine.EngineCloudXInterface.Update () [0x00025] in <b7143c5935a64e91bef20efb127a4c6d>:0 
```

The log itself can be seen from the 7.25 build.

### How often does it happen?

Sometimes

### Does the bug persist after restarting Neos?

No

### Neos Version Number

2021.7.25.489, 2021.8.2.936

### What Platforms does this occur on?

Windows

### Did this work before?

I Don't Know

### Additional context

You can see the details here. I'll add the log as soon as I get it.
https://discord.com/channels/402159838827905024/655375765596274756/874295034797752322

### Reporters

kazu, rhenium

**Comment by kazu0617** [comment]
![Screenshot_20210811-082831563 (1)](https://user-images.githubusercontent.com/6259214/128947796-d49fa352-6d38-4608-8f68-c491ef63fff7.jpg)
![unknown-4](https://user-images.githubusercontent.com/6259214/128947803-5ac7af3d-b2a5-499a-99f3-b797bb97fb67.png)
![unknown-4](https://user-images.githubusercontent.com/6259214/128947804-873c8b32-7765-4308-88c4-6a278bcfebf5.png)


I received a same problem from esnya.

**Comment by orange3134** [comment]
I'm having a similar problem.
I can provide a log file if needed.
![キャプチャ](https://user-images.githubusercontent.com/56525091/128953239-d66c7fc9-5e8c-4707-8a6c-d3908f78b39d.PNG)

**Comment by rabbuttz** [comment]
same here
[DESKTOP-CB6HJR0 - 2021.8.2.936 - 2021-08-10 01_40_41.log](https://github.com/Neos-Metaverse/NeosPublic/files/6966900/DESKTOP-CB6HJR0.-.2021.8.2.936.-.2021-08-10.01_40_41.log)

This may not be relevant, but I fixed it by changing the language setting to English. However, this solution had no effect on rhenium. It may have been a fluke. Just to be sure, I have attached the log file after the bug was fixed with the English setting.

[DESKTOP-CB6HJR0 - 2021.8.2.936 - 2021-08-10 16_44_15.log](https://github.com/Neos-Metaverse/NeosPublic/files/6966965/DESKTOP-CB6HJR0.-.2021.8.2.936.-.2021-08-10.16_44_15.log)

**Comment by AshtonSparx** [comment]
![image](https://user-images.githubusercontent.com/53165730/129175344-4a944bd0-92a1-45a9-bea8-8b82285f55dc.png)

Also getting this error too. The log is absolutely massive

**Comment by Frooxius** [comment]
Thank you for the report! I have a fix coming for this in the next build, it should be out tomorrow.

**Comment by Frooxius** [comment]
This should be fixed in 2021.8.12.1293, please let me know if you run into any more issues!
